converter/src/test/resources/finc-solr-schema.xml

```diff
@@ -0,0 +1,363 @@
+<?xml version="1.0" ?>
```

Comment:
Is it OK for us to include this file here? I think, UBL should at least consent to this.